sympy/printing/pycode.py

```diff
@@ -296,6 +296,14 @@ def _print_Sum(self, expr):
     def _print_ImaginaryUnit(self, expr):
         return '1j'
 
+    def _print_KroneckerDelta(self, expr):
+        a,b = expr.args
+
+        return '1 if {a} == {b} else 0'.format(
```

Comment:
Should this be parenthesized so that it works properly in larger expressions?

Reply:
You're right, added parens.